src/utils/vector_math_test.cc

```diff
@@ -0,0 +1,189 @@
+#include <catch2/catch_all.hpp>
+#include "vector_math.h"
+
+#define NOT_APPLICABLE 0.0
+
+TEST_CASE("calculateLinePointDistance calculates distance to infinite line", "[Geometry][LinePoint]")
```

Comment:
It would be nice with a structured way of selecting which tests to run, e.g.:
* Run all unit tests
* Run unit tests for my_file.cc

Perhaps using sections?
I thought tags would work, but couldn't get them working using ctest.

If you prefer pushing this back, let's open a ticket.

Reply:
I am missing an option to `catch_discover_tests` from https://github.com/catchorg/Catch2/blob/devel/docs/cmake-integration.md

`ADD_TAGS_AS_LABELS`

I don't think I can do that and tell it to add test filename tags though.